tests/acceptance/test_acceptance/test_ups.py

```diff
@@ -0,0 +1,148 @@
+import json
+
+import pytest
+
+from tests.acceptance.helpers import ENDPOINT_DECIDE, ENDPOINT_LOOKUP, ENDPOINT_SAVE
+from tests.acceptance.helpers import create_and_validate_request_and_response
+
+
+expected_flag_key = """
+    {
+      "variationKey": "variation_1",
+      "enabled": true,
+      "ruleKey": "feature_2_test",
+      "flagKey": "feature_2",
+      "userContext": {
+        "userId": "matjaz",
+        "attributes": {
+          "attr_1": "hola"
+        }
+      },
+      "reasons": []
+    }
+"""
+
+expected_after_saved_variation = """
+    {
+      "variationKey": "variation_2",
+      "enabled": true,
+      "ruleKey": "feature_2_test",
+      "flagKey": "feature_2",
+      "userContext": {
+        "userId": "user1",
+        "attributes": {
+          "attr_1": "hola"
+        }
+      },
+      "reasons": []
+    }
+"""
+
+
+@pytest.mark.parametrize(
+    "flag_key, expected_response, expected_status_code", [
+        ("feature_2", expected_flag_key, 200)
+    ],
+    ids=["valid case"])
+def test_ups__feature(session_obj, flag_key, expected_response, expected_status_code):
+    """
+    Test validates:
+    Same decide is returned after calling multiple times.
+    ...
+    :param session_obj:
+    :param flag_key:
+    :param expected_response:
+    :param expected_status_code:
+    """
+    payload = """
+        {
+          "userId": "matjaz",
+          "decideOptions": [
+              "ENABLED_FLAGS_ONLY"
+          ],
+          "userAttributes": {"attr_1": "hola"}
+        }
+    """
+
+    params = {"keys": flag_key}
+    for _ in range(5):
+      resp = create_and_validate_request_and_response(ENDPOINT_DECIDE, 'post', session_obj, payload=payload,
+                                                      params=params)
+
+      assert json.loads(expected_response) == resp.json()
+      assert resp.status_code == expected_status_code, resp.text
+      resp.raise_for_status()
+
+
+@pytest.mark.parametrize(
```

Comment:
Same here. No parameterization needed.
Ps: Unless you are expecting to add more test variations later